Vendor.API/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Host=localhost;Database=ILP_Backroads;Username=postgres;Password=root;"
+    "DefaultConnection": "Host=localhost;Database=ILP_Backroads;Username=postgres;Password=Password@1;"
```

Comment:
Keep a common db password or connection string do not push individual local system specific changes to PR